tasks/styles.js

```diff
@@ -8,7 +8,7 @@ const autoprefixer = require('gulp-autoprefixer');
 const gulpIf = require('gulp-if');
 const cleanCss = require('gulp-clean-css');
 const rename = require('gulp-rename');
-const sass = require('gulp-sass')(require('node-sass'));
+const sass = require('gulp-sass')(require('dart-sass'));
```

Comment:
You can see in the updated test data (`basic-expanded.css` below) that the only discernible difference between the output of `node-sass` and `dart-sass` is the order in which multiple selectors separated by commas and the fact `dart-sass` keeps related blocks together (that is, without a blank line in between). Neither of these changes have a functional impact on the way a browser interprets the CSS.